src/api/apps/articles/model/save.coffee

```diff
@@ -1,6 +1,7 @@
 _ = require 'underscore'
 _s = require 'underscore.string'
 db = require '../../../lib/db'
+{ amqp } = require '../../../lib/amqp'
```

Comment:
**question (non-blocking):** Minor — are the imports on L4 and L21 necessary in this file?

(I think there was a refactor since these were added)